2022/incoming/time-to-open-presents.pod

```diff
@@ -0,0 +1,170 @@
+Author: oodler@cpan.org
+Title: Wake up! Time to open presents! 
+Topic: $SIG::ALRM, signal handlers, alarm , Try::ALRM
+
+=encoding utf8
+
+=head1 Wake up! Time to open presents!
+
+Clark W. Grisman faced a dilemma, it was the night before
```

Comment:
Who is "Clark W. Grisman"? Seems a famous reference but it could help to introduce him (quickly).

Reply:
It's from National Lampoons, _Christmas Vacation_. His name is actually Clark W. Griswold (played by Chevy Chase), but is called _Grisman_ near the end by a package driver delivering his Jelly of the Month Club Christmas bonus from his company at the last hours on Christmas Eve. I skipped over a lot of stuff and made a lot of references to the movie, but if you see it once you'll get pretty much everything immediately.

Maybe I can mention it is loosely based off of _National Lampoons, Christmas Vacation_?

Reply:
Not critical, going to close this